app_dart/lib/src/foundation/utils.dart

```diff
@@ -179,6 +179,20 @@ Future<List<Target>> getTargetsToRun(Iterable<Target> targets, List<String?> fil
   return targetsToRun;
 }
 
+/// Returns `true` if [ciYaml.postsubmitTargets] should be ran during presubmit.
+///
+/// This is an **experimental** feature that is only enabled for the
+/// `flutter/engine` repository if a specific label is present on the PR.
+bool includePostsubmitAsPresubmit(CiYaml ciYaml, PullRequest pullRequest) {
+  if (ciYaml.slug != Config.engineSlug) {
+    return false;
+  }
+  if (pullRequest.labels?.any((label) => label.name.contains('run-postsubmit-as-presubmit')) ?? false) {
```

Comment:
Does this fit with the current label nomenclature? (I don't know the answer)

Reply:
I have no idea. Let's go with `test: all`.

Reply:
No clue! Changed to `test: all`.